src/Nethermind/Nethermind.Merge.Plugin/Synchronization/ChainLevelHelper.cs

```diff
@@ -28,7 +28,7 @@ namespace Nethermind.Merge.Plugin.Synchronization;
 
 public interface IChainLevelHelper
 {
-    BlockHeader[]? GetNextHeaders(int maxCount, long maxHeaderNumber);
```

Comment:
Small:
- Naming, can we have something easier to decipher? Dunno maybe `skipLastBlockCount`?
- Maybe should be 0 by default?

Reply:
Yea, my bad. Intellij refactor.